devicetypes/heltun/he-relay.src/he-relay.groovy

```diff
@@ -0,0 +1,38 @@
+/**
+ * 
+ *
+ *  Copyright 2021 Sarkis Kabrailian
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ */
+metadata {
+	definition (name: "HE-RELAY", namespace: "HELTUN", author: "Sarkis Kabrailian", cstHandler: true, ocfDeviceType: "oic.d.switch") {
```

Comment:
What about renaming "HE-RELAY" to "Heltun Child Relay" ? That would be more consistent and less confusing.
(make sure to change it in the main DTH too -> `addChildDevice()`)